docs/UserGuide.md

```diff
@@ -78,7 +78,9 @@ Adds a food item to the food inventory.
 
 Format: `add d/DESCRIPTION e/EXPIRY_DATE q/QUANTITY [p/PRIORITY] [t/TAG]…​`
 
-* Adds a food item based on its description and expiry date.
+* Adds a food item based on its description, expiry date and tags.
+* A food item with the same description, expriy date and tags as another food item is considered a duplicate.
```

Comment:
Small typo for 'expiry', same for other lines in the UG

Reply:
Alright updated. Thanks.